core/ecschema-editing/src/test/Editing/ECClasses.test.ts

```diff
@@ -362,4 +362,164 @@ describe("Property creation tests", () => {
     property = await entity?.getProperty(propResult.propertyName!) as PrimitiveProperty;
     expect(property).to.be.undefined;
   });
+
+  it("CustomAttribute defined in same schema, instance added to class successfully.", async () => {
```

Comment:
Add tests where you try to add a custom attribute to a class or property that does not exist.  Also add a test where you try to add a custom attribute to a schema item that does not support custom attributes (like KOQ)

Reply:
Sorry, need one more test....

Reply:
New tests added.